internal/scribe/src/samsara/scribe/core.clj

```diff
@@ -1,10 +1,30 @@
 (ns samsara.scribe.core
-  (:require [samsara.scribe.serializer.json :as json]))
+  (:require [samsara.scribe.serializer.json :as json]
+            [samsara.scribe.serializer.edn :as edn]
+            [samsara.scribe.serializer.nippy :as nippy]
+            [samsara.scribe.serializer.fressian :as fressian]
+            [samsara.scribe.serializer.transit :as transit]))
 
 
 (defmulti scribe :type)
 
 
 (defmethod scribe :json
   [config]
-  (json/make-json-scribe config))
+  (json/make-json-scribe (dissoc config :type)))
+
+(defmethod scribe :edn
+  [config]
+  (edn/make-edn-scribe (dissoc config :type)))
+
+(defmethod scribe :nippy
+  [_]
+  (nippy/make-nippy-scribe))
```

Comment:
I think also the nippy scriber should take a `config`.
there are several things which could be configured as [specified in the doc](http://ptaoussanis.github.io/nippy/taoensso.nippy.html#var-freeze) (like encription, compression etc).